README.md

```diff
@@ -0,0 +1,61 @@
+# BASiCS
+Bayesian Analysis of Single-Cell Sequencing Data is an integrated Bayesian hierarchical model where:
+
+- Cell-specific normalization constants are estimated as part of the model parameters.
+- Technical variability is quantified based on spike-in genes that are artificially introduced to each analysed cells lysate.
+- The total variability of the expression counts is decomposed into technical and biological components.
+
+## Installation
+
+Get the released version from CRAN:
+
+```R
+install.packages("BASiCS")
```

Comment:
Looks like BASiCS is not on CRAN. Is there a burocraticy process to fullfill before having a package available there?